blog/vscode-remote-development-setup-and-usage.mdx

```diff
@@ -0,0 +1,263 @@
+---
+title: "VSCode Remote Development: Setup and usage"
+authors: [gaetan]
+date: 2023-05-02T12:00
+hide_table_of_contents: true
+---
+
+# VSCode Remote Development: Setup and usage
+
+Every developer has their own preferences for their setup, whether it be
+software or hardware. I’ve been a remote developer since 2018, and since then
+I’ve tried various ways to organize my workspace at home, that would fit both my
+work and personal times.
+
+A typical remote developer kit will often include a laptop and one or more
```

Comment:
```suggestion
A typical remote developer kit will often include a laptop or desktop and one or more
```

Reply:
Just to be clear, I meant a company-provided remote developer kit. That said, I'm not sure if your review still applies, although this sentence could be rephrased to specify what I meant.